task/buildah/0.2/buildah.yaml

```diff
@@ -107,6 +107,12 @@ spec:
     description: Additional key=value labels that should be applied to the image
     type: array
     default: []
+  - name: PRIVILEGED_NESTED
+    description: Whether to enable privileged mode, it will add --privileged to podman
+      run and --security-opt=label=disable --cap-add=all --device=/dev/fuse to buildah
```

Comment:
Nitpick: let's remove the part about adding `--privileged` to podman. It's an implementation detail that users don't care about, and it's not true for the non-remote task

Reply:
Updated.